richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/hyperlink/HyperlinkOps.java

```diff
@@ -0,0 +1,41 @@
+package org.fxmisc.richtext.demo.hyperlink;
+
+import org.fxmisc.richtext.model.SegmentOpsBase;
+
+public class HyperlinkOps<S> extends SegmentOpsBase<Hyperlink<S>, S> {
```

Comment:
Just FYI. The updated version doesn't use `SegmentsOpsBase` It seems that even the empty segment object needs to be able to update its style because of what I mention in [my comment to your previous one](https://github.com/TomasMikula/RichTextFX/pull/550#discussion_r132587029). This is why I'm proposing #567.